### ISSUE TITLE
Sometimes the bot skips certain messages and does not delete them?

### ISSUE THREAD

**Issue by Solarris0111** [riking/AutoDelete#24]
Let's say I have an auto delete set at 3 messages and at 1 minute.
And I type this:
A B C D E F
Sometimes it will keep random parts of the message and never delete them even after the time has expired.
So after 10 minutes, it might read this:
C E
Any fix?

**Comment by riking** [comment]
Is this still happening?

**Comment by Edsonzhu** [comment]
> Is this still happening?

Yes. Also I set the bot to delete after 20h and it doesn't seem to delete
